generic_config_updater/gu_common.py

```diff
@@ -2,13 +2,15 @@
 import jsonpatch
 from jsonpointer import JsonPointer
 import sonic_yang
+import syslog
 import subprocess
 import yang as ly
 import copy
 import re
 from enum import Enum
 
 YANG_DIR = "/usr/local/yang-models"
+SYSLOG_IDENTIFIER = "GenericConfigUpdater"
```

Comment:
>GenericConfigUpdater [](http://example.com/codeflow?start=21&length=20)

Is this ident too long? The default will be process name such as `config`, is it good enough since you have add `title` to diffentiate.

Reply:
Also I don't think we should use `config` since this library might be used with `REST` APIs like was requested in some of the earlier meetings with SONiC Community